.node-version

```diff
@@ -1 +1 @@
-8.5.0
+10.12.0
```

Comment:
As of 2018-10-30, Node 10.x is now in [Active Long-Term Support](https://github.com/nodejs/Release#release-schedule).